buildSrc/src/main/groovy/nva.identity.service.gradlelint.gradle

```diff
@@ -0,0 +1,7 @@
+plugins{
```

Comment:
The nebula-lint plugin can only be applied on the root project. Therefore I created a plugin to be applied to the root project